deployment/environment/web/sdk/client/client.go

```diff
@@ -202,7 +202,11 @@ func (c *client) CreateJobDistributor(ctx context.Context, in JobDistributorInpu
 		feedsManager := success.GetFeedsManager()
 		return feedsManager.GetId(), nil
 	}
-	return "", fmt.Errorf("failed to create feeds manager")
+	if err, ok := response.GetCreateFeedsManager().(*generated.CreateFeedsManagerCreateFeedsManagerSingleFeedsManagerError); ok {
+		msg := err.GetMessage()
+		return "", fmt.Errorf("failed to create feeds manager: %v", msg)
+	}
+	return "", fmt.Errorf("failed to create feeds manager: %v", response.GetCreateFeedsManager().GetTypename())
```

Comment:
can the response be nil at any point here?

Reply:
I think I found a better way of handling all these: converting to `status.Status` then handling the `Code`/`Message` values: https://github.com/grpc/grpc-go/blob/master/Documentation/rpc-errors.md

Reply:
actually that won't work, the error responses aren't returned as actual errors... we'll just have to write custom cases

Reply:
We have [custom errors](https://github.com/smartcontractkit/chainlink-common/blob/214b4927ce3458c4b6a9a0f31dddda38b582096b/pkg/loop/internal/net/errors.go#L53) for loops that know how to translate to grpc and [still match via errors.Is](https://github.com/smartcontractkit/chainlink-common/blob/214b4927ce3458c4b6a9a0f31dddda38b582096b/pkg/loop/internal/net/errors.go#L69-L71)